templates/docs/examples/patterns/equal-height-row/4-items-per-column-responsive.html

```diff
@@ -0,0 +1,139 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Equal height row / Four items per column and full-screen divider{% endblock %}
+
+{% block standalone_css %}patterns_equal-height-row{% endblock %}
+
+{% block content %}
+<div class="p-equal-height-row has-divider-3">
+  <div class="p-equal-height-row__col">
+    <div class="p-equal-height-row__item">
+      <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+        <img class="p-image-container__image"
+            src="https://assets.ubuntu.com/v1/c4176cd9-Kernelt%20industries-80%201.jpg"
+            alt=""
+            width="250">
+      </div>
+    </div>
+
+    <div class="p-equal-height-row__item">
+      <hr class="is-highlighted u-hide--medium u-hide--small">
+
+      <h5 class="p-heading--5">
+        Continuous security
+      </h5>
+    </div>
+
+    <p class="p-equal-height-row__item">
+      Imagine a world where every device is trustworthy. We designed every
+      aspect of Ubuntu Core to create the most secure embedded Linux ever, with
+      a 10 year LTS commitment.
+    </p>
+
+    <div class="p-equal-height-row__item">
+      <a href="#">
+        <p>
+          Learn more about Ubuntu security&nbsp;›
```

Comment:
Nit: use HTML entity for the right chevron here and anywhere else it is found
```suggestion
          Learn more about Ubuntu security&nbsp;&rsaquo;
```